x/wasm/types/exported_keepers.go

```diff
@@ -53,6 +53,9 @@ type ContractOpsKeeper interface {
 
 	// SetContractInfoExtension updates the extension point data that is stored with the contract info
 	SetContractInfoExtension(ctx sdk.Context, contract sdk.AccAddress, extra ContractInfoExtension) error
+
+	// SetAccessConfig updates the access config of a code id.
+	SetAccessConfig(ctx sdk.Context, codeID uint64, config AccessConfig) error
```

Comment:
`NewWasmProposalHandlerX` takes `ContractOpsKeeper` as argument, not really sure how have access to this method without exporting it (not really sure if a private method makes sense on an interface)

Reply:
good point here. If there is a simple way to keep it private and expose to the proposal handler, maybe @alpe can make a quick PR to show how. Otherwise, I would be fine to merge like this.